.agents/skills/qodo-pr-resolver/resources/providers.md

```
@@ -0,0 +1,1399 @@
+- Create an App Password: Bitbucket → **Settings → App passwords**
+  - Required scopes: **Repositories: Read**, **Pull requests: Read, Write**
+- **Qodo config** (`~/.qodo/config.json`) — store credentials persistently:
+  ```json
+  {
+    "BB_USERNAME": "your-bitbucket-username",
+    "BB_APP_PASSWORD": "your-app-password"
+  }
+  ```
+  These examples support **Bitbucket Cloud only**. Bitbucket Server and Data Center use different REST routes and project/repository addressing, so do not point these commands at a self-hosted instance.
+- **Load configuration** (existing environment variables take precedence):
+  ```bash
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)